tests/test_sliderule.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+import geopandas as gpd
+import pytest
+
+import coincident
+
+# import sliderule
+# sliderule.init(verbose=True)
+
+network = pytest.mark.network
+
+
+def test_gdf_to_sliderule_polygon():
+    gf = gpd.GeoDataFrame(
+        geometry=gpd.points_from_xy(
+            [-106.46953038438801, -106.47209604357627],
+            [38.781089777057986, 38.78292421778072],
+            crs="EPSG:4326",
+        )
+    )
+    poly = coincident.io.sliderule._gdf_to_sliderule_polygon(gf)
+    assert isinstance(poly, list)
+    assert len(poly) == 5
+
+
+@network
+@pytest.mark.skip(reason="https://github.com/uw-cryo/coincident/issues/35")
```

Comment:
disabling this for now, as I can't get these tests to run consistently locally or in CI... https://github.com/uw-cryo/coincident/issues/35